packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/primative-types.test.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  ControllerSpec,
+  post,
+  requestBody,
+  getControllerSpec,
+} from '../../../../../';
+import {ContentObject, SchemaObject} from '@loopback/openapi-v3-types';
+import {Class} from '@loopback/repository';
+import {expect} from '@loopback/testlab';
+
+// I will add all shortcut decorators if we think they are valuable
```

Comment:
IMO it's good to add them all, and when we intend to use DI, they can be refactored.